cmd/admin-bucket-remote-add.go

```diff
@@ -183,18 +189,22 @@ func fetchRemoteTarget(cli *cli.Context) (sourceBucket string, bktTarget *madmin
 	if !madmin.ServiceType(serviceType).IsValid() {
 		fatalIf(errInvalidArgument().Trace(serviceType), "Invalid service type. Valid option is `[replication]`.")
 	}
-
+	bandwidthStr := cli.String("bandwidth")
+	bandwidth, e := humanize.ParseBytes(bandwidthStr)
+	fatalIf(probe.NewError(e).Trace(bandwidthStr), "Unable to parse quota")
+	fmt.Printf("bandwidth:%d\n", bandwidth)
```

Comment:
isn't this an optional flag? You might want to check if cli.IsSet("bandwidth") and set it in BucketTarget if this flag is set.